### PR TITLE
Initial docs for EIP-6963 support

### PR DESCRIPTION
**Pull request by alexandratran**
Add concept and how-to page for wallet interoperability and EIP-6963 support. This is meant to be initial documentation to be expanded upon later.

previews:
- https://docs.metamask.io/eip-6963/wallet/concepts/wallet-interoperabilty/
- https://docs.metamask.io/eip-6963/wallet/how-to/discover-multiple-wallets/

### CODE REVIEW

File: wallet/concepts/apis.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ This API is specified by [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193), an
 request users' Ethereum accounts, read data from blockchains the user is connected to, suggest
 that the user sign messages and transactions, and more.
 
+:::note
+MetaMask supports [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963), which introduces an
+alternative discovery mechanism to the `window.ethereum` injected provider, enabling dapps to
```
Comment (jiexi) [inline wallet/concepts/apis.md:27]: i think we should mention somewhere that this is the new and preferred pattern for provider discovery

File: wallet/concepts/apis.md [hunk 2]
```diff
@@ -22,6 +22,13 @@ This API is specified by [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193), an
...
+support [wallet interoperability](wallet-interoperabilty.md) and discover multiple injected wallet
+providers in a user's browser.
```
Comment (jiexi) [inline wallet/concepts/apis.md:29]: enable wallet interoperability by ?
```suggestion
support [wallet interoperability](wallet-interoperabilty.md) by enabling the discovery of multiple injected wallet
providers in a user's browser.
```

idk, reads a little oddly to me. feel free to ignore

File: wallet/concepts/wallet-interoperabilty.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  <video width="100%" controls>
+    <source src="/eip-6963/eip-6963.mov" />
+  </video>
+</p>
+
+You can add support for EIP-6963 in one of the following ways:
+
+- [Set up MetaMask SDK](../how-to/connect/set-up-sdk/javascript/index.md) in your dapp.
+  The SDK supports EIP-6963 by default, and we recommend using this method.
+- [Directly update your dapp code](../how-to/discover-multiple-wallets.md) to support EIP-6963.
```
Comment (jiexi) [inline wallet/concepts/wallet-interoperabilty.md:24]: i feel like a lot of what is said on this linked page is the same as what is already on this page itself. Would it make sense to have the discover-multiple-wallets page simply link back this page instead?

File: wallet/concepts/apis.md (hunk 1 above)
Comment (jiexi) [reply wallet/concepts/apis.md:27]: and/or emphasize that `window.ethereum` injection is deprecated (but in reality, we won't be removing this for years if ever)

File: wallet/concepts/wallet-interoperabilty.md (hunk 3 above)
Comment (alexandratran) [reply wallet/concepts/wallet-interoperabilty.md:24]: The `discover-multiple-wallets` page does link back to this page (the first sentence links to "wallet interoperability"). It does contain a lot of the same intro content, but it's more of placeholder for when we have more thorough instructions for directly updating the dapp code.

**Review by jiexi — APPROVED** [review]
🤩

File: wallet/concepts/apis.md
Comment (mjsmike62) [inline wallet/concepts/apis.md:1]: Line 26 & 27, sentence is a little lengthy. May want to break up into separate sentences.

File: wallet/how-to/connect/detect-metamask.md
Comment (mjsmike62) [inline wallet/how-to/connect/detect-metamask.md:1]: For lines 26-29, this content is a little lengthy. Can you break it up into multiple sentences?

**Review by mjsmike62 — APPROVED** [review]
LGTM. Just a couple of minor edits.

**Pull request merged**